sync/class.jetpack-sync-settings.php

```diff
@@ -19,6 +19,7 @@ class Jetpack_Sync_Settings {
 		'disable'                 => true,
 		'render_filtered_content' => true,
 		'post_meta_whitelist'     => true,
+		'user_meta_whitelist'     => true,
```

Comment:
are sync settings per site on a multisite install?

Reply:
They are per site and not per network.